wrap/tests/expected/MyFactorPosePoint2.m

```diff
@@ -1,36 +0,0 @@
-%class MyFactorPosePoint2, see Doxygen page for details
```

Comment:
Where did all the expected matlab files go?

Reply:
`expected-matlab`